odata-protocol/11.4 Data Modification.md

```diff
@@ -364,7 +367,11 @@ property `$value`.
 Updating a dependent property that is tied to a key property of the
 principal entity through a referential constraint updates the
 relationship to point to the entity with the specified key value. If
-there is no such entity, the update fails.
+the entity set for that entity cannot be determined or does not contain
+such an entity, the update fails. The entity set can be determined, among
+other possibilities, by containment navigation properties or navigation property bindings in the
+service metadata, or by `@context` or `@id` control information in the request
+payload.
```

Comment:
```suggestion
other possibilities, by containment or navigation property binding,
or by control information in the request payload.
```
I'd prefer to keep specifics of the JSON Format out of the Protocol specification.